### ISSUE TITLE
/mask /gmask not properly working when excluding blocks

### ISSUE THREAD

**Issue by Craftbeef** [IntellectualSites/FastAsyncWorldEdit#1412]
### Server Implementation

Paper

### Server Version

1.17.1

### Describe the bug

tried to use //br sphere 0 10 with /mask !red_wool,netherrack,gray_concrete   --> i had a boat i wanted to remake and wanted to replace everything except for the hull of the boat made of the materials mentioned after !

### To Reproduce

1. equip a brush, for example "//br sphere 0 10"
2. equip a Mask or Global Mask for example "/mask !red_wool,netherrack,gray_concrete"
3. apply brush to preferred area

### Expected behaviour

To set mask and global masks to both parameters being included when the exclude ( ! ) is added to the command. 
In simple terms: to only edit the blocks which are not being excluded by the mask.

### Screenshots / Videos

![image](https://user-images.githubusercontent.com/26029598/141655243-ee1b8001-6d3d-49ca-a47d-4f159956eedc.png)

### Fawe Debugpaste

https://athion.net/ISPaster/paste/view/26e6436f843741e8a03900d9bf34e827

### Fawe Version

FastAsyncWorldEdit-Bukkit-1.17-396

### Checklist

- [X] I have included a Fawe debugpaste.
- [X] I am using the newest build from https://ci.athion.net/job/FastAsyncWorldEdit-1.17/ and the issue still persists.

**Comment by NotMyFault** [comment]
I'm unable to replicate what you described based on the steps provided.
![](https://i.imgur.com/jVHPOYF.jpg)
If you can reproduce that on an up to date version of Paper and Fawe, please provide the exception the console throws when notifying you ingame.

**Comment by Craftbeef** [comment]
Hi, I have done some further testing this time of a local copy of the server this is happening on.
The results were a plugin conflict between FAWE, seemingly CoreProtect, and WorldGuardExtraFlags.
As suggested by your plugin I've added the needed parameters to the allowed plugins list which resolved the issue.

The specific lines in question:
`  "net.goldtreeservers.worldguardextraflags.wg.wrappers.v7.WorldEditFlagHandler"
   "net.coreprotect.worldedit.CoreProtectLogger" `
